app/api/routes.py

```diff
@@ -13,26 +13,34 @@
 from prometheus_client import Counter, Summary
 import app.utils as utils
 from os import environ
-
+from flask_cors import cross_origin
 
 # Metrics
 failures_counter = Counter('my_failures', 'Number of exceptions raised')
 latency_summary = Summary('request_latency_seconds', 'Length of request')
 
 logger = utils.setup_logger('routes_logger')
 
+ALLOWED_ORIGINS = [
+    'http://localhost:3000',
+    r"https:\/\/(www\.)?operationcode\.org",
+    r"https:\/\/(.*\.)?operation-code(-.*)?\.now\.sh"
+]
+
 
 # Routes
 @latency_summary.time()
 @failures_counter.count_exceptions()
 @bp.route('/resources', methods=['GET'], endpoint='get_resources')
+@cross_origin(origins=ALLOWED_ORIGINS)
```

Comment:
Do we need CORS headers for the GET endpoints? Or is this just to keep things consistent with all routes? I honestly don't know the answer to this, but I know that there are some resources making their way to the frontend right now (20 of them, right?)

Reply:
Correct. The resources are in fact available without CORS headers when you first visit the resources route. However, you won't be able to fetch additional resources from the client-side without CORS headers once you are on that route(e.g. when navigating to pages other than the first resource page). At least, that is my initial thought on how to handle data fetching.

There might be a way to go about it without enabling CORS for GET endpoints and I am looking into Next.js docs to see if it would be possible. Maybe should we circle back to this issue after some research?

Reply:
That sounds reasonable, I'm not super worried about it. Is there any way to test for this locally besides just looking at the headers?